### ISSUE TITLE
[Due for payment 2026-03-02] [$250] Screen Reader: Many Pages: The informative icon before the error message is not announced

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77256]
## Action Performed:
1. Using Android TalkBack or iOS VoiceOver, open the Expensify app
2. Swipe to the 'Continue' button and double-tap to activate without entering the phone or email
3. Note that the error message appeared on the screen
4. Swipe and try to focus the error icon before the error message.

Other Occurrences:
1. On Sign In - Enter Code
2. Onboarding: all fields in the flow
3. On Inbox: the field 'Enter your boss's email'
4. On Profile - Display Name
5. On Profile - Date of Birth
6. On Profile - Address
7. On Profile - Legal Name
8. On Profile - Verify Email
9. On Security - Merge Accounts
10. On Security - Merge Accounts - Enter Code
11. On Security - Close Account
12. On Wallet - Add Bank Account
13. On Create Expense - Confirm Details
14. On Save the World - I know a teacher
15. On Create Workspace

## Expected Result:
The informative icon should be announced, e.g.:
- 'error'

## Actual Result:
The informative icon before the error message is not announced.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Use the accessibilityLabel to announce the error message together with the icon, e.g.:
- 'error <error_text>'
https://appt.org/en/docs/react-native/samples/accessibility-label

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/92ed084a-ae41-4d45-baa5-2987f3f2f01f)

https://github.com/user-attachments/assets/5162bded-c45e-441d-b7f0-5394ae4f7107

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005013
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @jasperhuangg</details>
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022020939655765041348</li>
        <li>Upwork Job ID: 2020939655765041348</li>
        <li>Last Price Increase: 2026-02-09</li>
    </ul>
</details>
